template/haskell-http/function/src/Handler.hs

```diff
@@ -9,17 +9,12 @@ import Data.Text.Encoding.Error ( UnicodeException (..))
 import Data.ByteString.Lazy (ByteString)
 import qualified Data.ByteString.Lazy as LBS
 import Network.Wai
+import qualified Text.PrettyPrint as T
 
-data Message = Message 
-  { statusCode :: Int
-  , content :: T.Text 
-  } deriving (Show, Eq, Generic)
-
-instance ToJSON Message
```

Comment:
Nice. As we talked, we probably want to make new types (Request/Response) but let's do that in a separate PR once you get this merged